test_cromwell.sh

```diff
@@ -93,7 +93,7 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
     echo "Building Cromwell"
     sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR=$(find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar")
+    CROMWELL_JAR=$(find "${RUN_DIR}"/cromwell/target/scala-2.* -name "cromwell-*.jar")
```

Comment:
Discussed in person, but to avoid having to update this in the future when Cromwell is updated from 2.11 to 2.x